content/en/docs/setup/platform-setup/amazon-eks/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Amazon EKS
+description: Steps to Configure Istio on Amazon EKS in AWS cloud.
+weight: 69
```

Comment:
Looking at the generated files, looks like you have added Amazon EKS as the last option, though everything else seems to be ordered alphabetically? Could you please adjust the weight so that the alphabetical order is preserved? https://deploy-preview-13822--preliminary-istio.netlify.app/latest/docs/setup/platform-setup/

Reply:
I see. I thought it was based on the PR timeline from each platform, sure I will change to make it ordered alphabetically.